modules/ROOT/pages/for-each-scope-concept.adoc

```diff
@@ -95,6 +88,15 @@ In case the message inside the For Each scope is persisted, not only the item in
 To avoid this issue you must first remove the rootMessage variable from the message before persisting it.
 ////
 
+=== Differences between For Each and Split Aggregate scopes
+
+There is an obvious similarity between the For Each and the xref:split-aggregate-scope.adoc[Split Aggregate scope]: both split the defined collection and process each element in it with the components within the scope. Also, in both cases, each route runs with the same initial context.
+The difference with these two scopes are:
+
+1. For Each works sequentially, while the Split Aggregate does it in parallel.
+1. The For Each does not modify the payload, while the Split Aggregate will output a collection of the output messages of each iteration.
+1. Error handling: since the first mentioned difference, For Each execution will be interrupted when an error is raised (and the Error Handler will be invoke), while Split Aggregate will process each route and then invoke the xref:intro-error-handlers.adoc[Error Handler], with the aggregation of the results and erros of all the routes.
```

Comment:
```suggestion
1. Error handling: since the first mentioned difference, For Each execution will be interrupted when an error is raised (and the Error Handler will be invoked), while Split Aggregate will process each route and then invoke the xref:intro-error-handlers.adoc[Error Handler], with the aggregation of the results and erros of all the routes.
```